src/main/java/org/gridsuite/study/server/networkmodificationtree/RootNodeInfoRepositoryProxy.java

```diff
@@ -126,11 +133,31 @@ public BuildStatus getBuildStatus(AbstractNode node) {
     }
 
     @Override
-    public void invalidateBuildStatus(AbstractNode node) {
+    public void invalidateBuildStatus(AbstractNode node, List<UUID> changedNodes) {
         RootNode rootNode = (RootNode) node;
         if (rootNode.getBuildStatus() == BuildStatus.BUILT) {
             rootNode.setBuildStatus(BuildStatus.BUILT_INVALID);
```

Comment:
You could call updateBuildStatus method here to avoid code duplication ?

Reply:
Done